### DISCUSSION TITLE
Chargemol interface

### DISCUSSION THREAD

**Discussion by juerghutter** [cp2k/cp2k#2598] [Team Posts]
I have been contacted by the lead developer of the Chargemol program (ddec.sourceforge.net). He asked if it would be possible to implement an interface to Chargemol that goes beyond Cube files. The interface would be through the wfx file format. A brief summary of wfx file information is found here:
http://aim.tkgristmill.com/wfxformat.html  

Here is a list of properties calculated by Chargemol (it will also work with AIMAII)

Functionality of the Chargemol program includes computing:
(1) net atomic charges -- for describing chemical states of atoms in materials as well as for approximately reproducing the electrostatic potential surrounding the material
(2) atomic spin moments -- for describing the magnetic ordering of atoms in materials
(3) bond orders -- accurate across diverse bonding types and diverse materials
(4) polarizabilities, dispersion coefficients (C6, C8, C9, C10), quantum Drude oscillator parameters  -- computed using MCLF method
(5) electron cloud parameters, and radial moments --  various radial moments are computed and the tail of each atom is fit to an exponential decay function
(6) atomic dipoles and quadrupoles
(7) bond order components -- the projection of bond order components onto individual orbitals shows which orbitals are bonding, anti-bonding, or non-bonding
(8) spdfg populations -- the s, p, d, f, and g subshell populations can be computed for each atom in a material as well as for individual orbitals
(9) projected density of states plots -- these provide a wealth of useful information and show the energy distribution of orbitals, spdfg contributions, and bond order components

Is somebody interested in such a small project. I assume it will not be too hard, especially as the developer would be available for questions and tests.

**Comment by DCM-Uni-Paderborn** [comment]
Dear Jürg, 

Hossam Elgabarty, who had already been using ddec charges, 
would be interested in that.  

Greetings, 
Thomas
